client/service_logs.go

```diff
@@ -44,7 +44,10 @@ func (cli *Client) ServiceLogs(ctx context.Context, serviceID string, options ty
 	}
 	query.Set("tail", options.Tail)
 
-	resp, err := cli.get(ctx, "/services/"+serviceID+"/logs", query, nil)
+	headers := map[string][]string{
+		"Accept": {"application/vnd.docker.raw-stream", "application/vnd.docker.multiplexed-stream"},
```

Comment:
I started looking at that as well (see https://github.com/moby/moby/pull/39812#discussion_r843314358), but

- They both have the same quality (weight), so it would depend on the implementation if it made a difference.
- It only matters if there's _multiple_ types to choose from (currently there's always only _one_ option, and it depends on if a TTY is enabled or not)